src/main/java/com/onarandombox/MultiverseCore/commandtools/MVCommandCompletions.java

```diff
@@ -66,4 +68,16 @@ private Collection<String> suggestMVWorlds(BukkitCommandCompletionContext contex
 
         return worlds;
     }
+
+    private Collection<String> suggestGamerules(BukkitCommandCompletionContext context) {
+
+        // Changes Collection of GameRules to Collection of Strings
+        Collection<String> gameRules = new ArrayList<>(Collections.emptyList());
+
+        for (GameRule<?> currentRule : GameRule.values()) {
+            gameRules.add(currentRule.getName());
+        }
```

Comment:
Possible to use streams to look cleaner, something like
```java
return Arrays.stream(GameRule.values()).map(GameRule::getName).collect(Collectors.toList());
```

Reply:
And maybe since gamerules is essentially fixed, we can consider making it a constant field.

Reply:
I like the idea of getting it from Bukkit as recently a lot of new gamerules are getting added.

Reply:
I mean still getting from bukkit, but only the first time when you start MV. I don't think Gamerules will change during runtime

Reply:
Ah, makes sense. I'll make a public list on startup, then return that in the command completion.

Reply:
Actually i just remembered, there is a `registerStaticCompletion` maybe can use that instead.